### PR TITLE
Azure Cosmos container with multiple containers

### PR DESCRIPTION
**Pull request by rahulpnath**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add a new template to create Azure Cosmos container with multiple containers

### CODE REVIEW

File: 101-cosmosdb-sql-multiple-containers/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
```
Comment (bmoore-msft) [inline 101-cosmosdb-sql-multiple-containers/azuredeploy.json:5]: add a defaultValue using uniqueString()

File: 101-cosmosdb-sql-multiple-containers/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      "type": "string",
+      "metadata": {
+        "description": "Cosmos Account Name"
+      }
+    },
+    "databaseName": {
```
Comment (bmoore-msft) [inline 101-cosmosdb-sql-multiple-containers/azuredeploy.json:11]: add a defaultValue

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
See the pipeline results for other changes than need to be made

**Comment by rahulpnath** [comment]
@bmoore-msft Thank you for pointing that out - Fixed them.

**Pull request merged**